src/electron/electron/core.cljs

```diff
@@ -119,8 +118,10 @@
                            (fn [part]
                              (. fs copy (path/join app-path part) (path/join static-dir part)))
                            ["css" "fonts" "icons" "img" "js"])))
-                export-css (. fs readFile export-or-custom-css-path)
-                _ (. fs writeFile (path/join static-dir "css" "export.css") export-css)
+                export-css (. fs readFile export-css-path)
+                _ (. fs writeFile (path/join static-dir "css" "export.css")  export-css)
+                custom-css (. fs readFile custom-css-path)
+                _ (. fs writeFile (path/join static-dir "css" "custom.css") custom-css)
```

Comment:
I believe this change breaks #5303. I think listing out the use cases for publish would help:

1. User defines export.css and custom.css and wants both included in their published graph. This is your use case
2. User defines export.css and custom.css and only wants export.css in their published graph. This use case was introduced with https://github.com/logseq/logseq/pull/5303
3. User defines only custom.css and wants it included in their published graph. Original workflow
4. User defines only export.css and wants it included in their published graph. Workflow supported by #5303
5. User defines no css files and their published graph just works. This use case is currently broken but I could fix it once this PR is done.

I don't see how use case 1 and 2 are compatible so we would need to ask the author if they are ok with that change and also see if it's used in the community. It [doesn't appear to be used much](https://cs.github.com/?scopeName=All+repos&scope=&q=path%3Aconfig%2Fexport.css)

Reply:
@QWxleA Would you be ok if the workflow you introduced in #5303 were replaced with use case 1 introduced with this PR?

Reply:
@pengx17 Do you know of any users that rely on the export.css introduced in #5303 and if they would be impacted by changing from use case 2 to use case 1 mentioned above?

Reply:
🤔 IMHO I think it is a bit troublesome and a bit confusing to introduce two separate css files to style the app. I do agree it feels more intuitive to follow the first use case.

Reply:
> 🤔 IMHO I think it is a bit troublesome and a bit confusing to introduce two separate css files to style the app. I do agree it feels more intuitive to follow the first use case.

From what I understand and propose: custom.css would be to style the app/graph, and export.css would strictly be to customize what the exported HTML graph looks like.

As of now, HTML exports still present a number of issues in regard to settings and components that shouldn't be visible in the export but need to be visible when editing pages in the app (see [Logseq custom.css for publishing](https://gist.github.com/solarkraft/b05e9058ca2cda40a4d6e03c52dff025), https://github.com/logseq/logseq/issues/6066, https://github.com/logseq/logseq/issues/4552).

Reply:
Sorry, I did not make myself clear. I am favoring your solution over the original one.

Reply:
@pengx17 ah, gotcha!

Reply:
@QWxleA If I don't hear back in a couple of days I'll be accepting this PR